modules/op-disabling-pipeline-templates.adoc

```diff
@@ -0,0 +1,44 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-disabling-pipeline-templates_{context}"]
+= Disabling resolver tasks and pipeline templates
+
+By default, the `TektonAddon` custom resource (CR) installs `resolverTasks` and `pipelineTemplates` resources along with {pipelines-shortname} on the cluster.
```

Comment:
Maybe we could mention `resolverStepActions` here.

Reply:
Good idea, I will note it and create a Jira for this, so that we do not hold this PR before the release. Thank you!

Reply:
Here is the Jira: https://issues.redhat.com/browse/RHDEVDOCS-6319